app/cli.php

```diff
@@ -7,209 +7,130 @@
 
-            return $database;
-        }
-
-        $dbAdapter = $pools
-            ->get($dsn->getHost())
-            ->pop()
-            ->getResource();
-
-        $database = new Database($dbAdapter, $cache);
-
-        $databases[$dsn->getHost()] = $database;
-
-        if ($dsn->getHost() === System::getEnv('_APP_DATABASE_SHARED_TABLES', '')) {
-            $database
-                ->setSharedTables(true)
-                ->setTenant($project->getInternalId())
-                ->setNamespace($dsn->getParam('namespace'));
-        } else {
-            $database
-                ->setSharedTables(false)
-                ->setTenant(null)
-                ->setNamespace('_' . $project->getInternalId());
-        }
-
-        $database
-            ->setMetadata('host', \gethostname())
-            ->setMetadata('project', $project->getId());
-
-        return $database;
-    };
-}, ['pools', 'dbForConsole', 'cache']);
-
-CLI::setResource('queue', function (Group $pools) {
-    return $pools->get('queue')->pop()->getResource();
-}, ['pools']);
-CLI::setResource('queueForFunctions', function (Connection $queue) {
-    return new Func($queue);
-}, ['queue']);
-CLI::setResource('queueForDeletes', function (Connection $queue) {
-    return new Delete($queue);
-}, ['queue']);
-CLI::setResource('queueForCertificates', function (Connection $queue) {
-    return new Certificate($queue);
-}, ['queue']);
-CLI::setResource('logError', function (Registry $register) {
-    return function (Throwable $error, string $namespace, string $action) use ($register) {
-        $logger = $register->get('logger');
-
-        if ($logger) {
-            $version = System::getEnv('_APP_VERSION', 'UNKNOWN');
-
-            $log = new Log();
-            $log->setNamespace($namespace);
-            $log->setServer(\gethostname());
-            $log->setVersion($version);
-            $log->setType(Log::TYPE_ERROR);
-            $log->setMessage($error->getMessage());
-
-            $log->addTag('code', $error->getCode());
-            $log->addTag('verboseType', get_class($error));
-
-            $log->addExtra('file', $error->getFile());
-            $log->addExtra('line', $error->getLine());
-            $log->addExtra('trace', $error->getTraceAsString());
-
-            $log->setAction($action);
-
-            $isProduction = System::getEnv('_APP_ENV', 'development') === 'production';
-            $log->setEnvironment($isProduction ? Log::ENVIRONMENT_PRODUCTION : Log::ENVIRONMENT_STAGING);
-
-            $responseCode = $logger->addLog($log);
-            Console::info('Usage stats log pushed with status code: ' . $responseCode);
-        }
-
-        Console::warning("Failed: {$error->getMessage()}");
-        Console::warning($error->getTraceAsString());
-    };
-}, ['register']);
+            Console::warning("Failed: {$error->getMessage()}");
+            Console::warning($error->getTraceAsString());
+        };
+    });
+
+$container->set($context);
```

Comment:
what about other resources we removed like dbforproject? and some queues?

Reply:
They are set in the `init.php` file